step-plans/step-plans-base-artefacts/src/main/java/step/artefacts/handlers/functions/autoscaler/TokenAutoscalingConfiguration.java

```diff
@@ -1,11 +1,15 @@
 package step.artefacts.handlers.functions.autoscaler;
 
+import io.kubernetes.client.openapi.models.V1ConfigMapList;
+
 import java.util.Map;
 
 public class TokenAutoscalingConfiguration {
 
     public Map<String, Map<String, String>> availableTokenPools;
 
+    public V1ConfigMapList availableConfigMapList;
```

Comment:
@rubij this V1ConfigMapList class is specific to the k8s driver and should not appear in this module, please refactor and remove the dependency from the pom.xml

Reply:
Done